src/main/java/searchengine/vsm/VectorSpaceModel.java

```diff
@@ -69,9 +77,17 @@ private void calculateTFIDF(List<TFIDFDocument> documents) {
     for (TFIDFDocument document : documents) {
       HashMap<String, Double> innerMap = new HashMap<>();
 
+      for (String term : document.getTF().keySet()) {
```

Comment:
Kommentar på hvad der foregår her og måske gem document.getTF() i en variabel i stedet for at kalde det sygt mange gange

Reply:
```suggestion
      // Find the maximum term frequency in the document
      for (String term : document.getTF().keySet()) {
```

Reply:
Det kan måske også gøres ved noget ala 
```
maximumFrequency = max(document.getTF().valueSet())
```